spec/unit/runner/variant-reassignment.spec.js

```diff
@@ -307,4 +309,62 @@ describe('Variant reassignment', () => {
       expect(matchingProductsVars.length).to.equal(0)
     })
   })
+
+  describe('ensure slug uniqueness', () => {
+    const testProductId = 'test-product-id'
+    let productsToAnonymize
+    let variantReassignments
+    beforeEach(() => {
+      productsToAnonymize = []
+      const productService = new ProductManager(utils.logger, {})
+      sinon.stub(productService, 'anonymizeCtpProduct').callsFake((product) => {
```

Comment:
Here you can use a simple `stub.returns(Promise.resolve())` and later in the code you can check `stub. callCount` and `stub.args` so you will get rid of redundant variable `productsToAnonymize `

Reply:
But isn't callCount only for spy? And if you have spy, then it calls the real method and you cannot fake the method?

Reply:
Try to print out stub object - I think that it extends spy functionality

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/37/commits/a0772affb546dc46fcda78c5d61e0c5eed5a615d